src/Photos.vue

```diff
@@ -69,10 +68,18 @@ export default {
 			loading: true,
 			svgplaceholder,
 			imgplaceholder,
-			videoplaceholder,
-			MapsInstalled
+			videoplaceholder
 		}
 	},
+	computed: {
+		mapsLink() {
+			if (isMapsInstalled) {
+				return 'apps/maps'
+			} else {
+				return 'https://apps.nextcloud.com/apps/maps'
```

Comment:
I'm wondering if we should rather redirect to the nextcloud apps settings ?
https://nc/settings/apps/integration/maps

Reply:
changed it